mount.go

```diff
@@ -143,37 +144,106 @@ func parseMountFlagsAndOptions(optionList []string) (int, string, error) {
 	return flags, strings.Join(options, ","), nil
 }
 
-func addMounts(mounts []*pb.Storage) ([]string, error) {
+func removeMounts(mounts []string) error {
+	for _, mount := range mounts {
+		if err := syscall.Unmount(mount, 0); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// storageDriversHandler is the type of callback to be defined to handle every
+// type of storage driver.
+type storageDriversHandler func(storage pb.Storage) (string, error)
+
+// storageDriversHandlerList lists the supported drivers.
+var storageDriversHandlerList = map[string]storageDriversHandler{
+	driver9pType:   storage9pDriverHandler,
+	driverBlkType:  storageBlkBlockDeviceDriverHandler,
+	driverSCSIType: storageSCSIBlockDeviceDriverHandler,
```

Comment:
Again, these names are really long. Maybe this is a sign the code should be broken into discrete packages to avoid `crazyLongNamesToAvoidCollidingInTheGlobalMainNamespace`?